### ISSUE TITLE
eventMgr (subscribe) - filter duplicate subscriptions

### ISSUE THREAD

**Issue by jp6rt** [jp6rt/sasutil#8]
dashboard.component.ts
screenshot: https://goo.gl/WkQ8LZ

```
// filter this to subscribe only once
this.notifications.eventMgr
.filter((event) => event.type === 'connectionStatusChange')
.subscribe((event) => {
	const { name, status } = event.data
	lo_.remove(this.connections, (c) => c.name === name),
	this.connections.push(this.setConnectionStatus(name, status))
})
```

**Comment by jp6rt** [comment]
Fix introduced another bug where 'connections' array is overlapped with previous state.

**Comment by jp6rt** [comment]
Fix reverted [95fe3b7] (https://github.com/jp6rt/sasutil/commit/ea601c153bc969a119148709abefa392eb4ae72b)
